themes/custom/essex/templates/content/node--localgov-newsroom--full.html.twig

```diff
@@ -73,14 +73,14 @@
 
 {%
   set classes = [
```

Comment:
remove indent from this line to match the following lines

Reply:
Fixed.